packages/rich-text-from-markdown/src/__test__/index.test.ts

```diff
@@ -80,7 +80,12 @@ describe('parses complex inline image markdown correctly', () => {
           nodeType: 'image',
           data: {},
         },
-        block(BLOCKS.PARAGRAPH, {}, text('\n      ')),
+        block(
+          BLOCKS.PARAGRAPH,
+          {},
+          text(`
+`),
```

Comment:
The previous expectations and this one are not the same I think. Do we expect any significant changes in the output?

Reply:
I checked some larger markdown files (e.g. contribution.md) and compared them with the current and the master functionality.
I wouldn't expect significant changes